src/main/java/algorithms/util/math/PrimeFactorsGenerator.java

```diff
@@ -0,0 +1,117 @@
+package algorithms.util.math;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class PrimeFactorsGenerator {
```

Comment:
![image](https://user-images.githubusercontent.com/60903744/120771708-2ed4e500-c528-11eb-9d07-df3fabc272eb.png)

Reply:
This is not a solution to the problem. This is just an algorithm for generating prime factors of a number. That's why it is in the math package.

Reply:
Oke, now I understand. I will review then.